### PR TITLE
[ci] wsu test refactor follow-up

### PR DESCRIPTION
**Pull request by sebsoto**
This PR includes:
- Re-adding dropped code to test framework 
- Removing an unnecessary global variable
- Silencing added test noise

### CODE REVIEW

File: internal/test/framework/framework.go [hunk 1]
```diff
@@ -91,12 +91,29 @@ func (f *TestFramework) getKubeClient() error {
 func (f *TestFramework) createWindowsVM() error {
 	// TODO: Add validation for the environment variables to see if they're properly populated or not
 	kubeconfig := os.Getenv("KUBECONFIG")
+	if kubeconfig == "" {
```
Comment (ravisantoshgudimetla) [inline internal/test/framework/framework.go]: I had a TODO here - https://github.com/openshift/windows-machine-config-operator/pull/113/commits/116103845216fb648b9ce342483dc6878952de4a#diff-5465ccc52b3a24461a98f029910cf60aR92, thanks for working on this @sebsoto

File: internal/test/framework/framework.go [hunk 2]
```diff
@@ -149,45 +150,49 @@ func (f *TestFramework) setupWinRMClient() error {
 // configureOpenSSHServer configures the OpenSSH server using WinRM client installed on the Windows VM.
 // The OpenSSH server is installed as part of WNI tool's CreateVM method.
 func (f *TestFramework) configureOpenSSHServer() error {
+	stdout := new(bytes.Buffer)
```
Comment (ravisantoshgudimetla) [inline internal/test/framework/framework.go]: Are you finding anything annoying in the output to silent it?

Comment (sebsoto) [reply internal/test/framework/framework.go]: This is the output on every test run
```
[ssoto@localhost windows-machine-config-operator]$ make run-wsu-ci-e2e-test 
hack/run-wsu-ci-e2e-test.sh
ssoto
network.operator.openshift.io/cluster patched (no change)

Name                           Version          Source           Summary                                               
----                           -------          ------           -------                                               
nuget                          2.8.5.208        https://onege... NuGet provider for the OneGet meta-package manager    




    Directory: C:\


Mode                LastWriteTime         Length Name                                                                  
----                -------------         ------ ----                                                                  
d-----       12/13/2019   2:53 AM                Temp                                                                  


=== RUN   TestWSU
```

File: internal/test/framework/framework.go (hunk 1 above)
Comment (sebsoto) [reply internal/test/framework/framework.go]: Forgot to take it out, updated with it removed

File: internal/test/framework/framework.go (hunk 2 above)
Comment (ravisantoshgudimetla) [reply internal/test/framework/framework.go]: SG

**Pull request merged**